lib/Sema/CSDiagnostics.cpp

```diff
@@ -2352,3 +2352,100 @@ bool InaccessibleMemberFailure::diagnoseAsError() {
   emitDiagnostic(Member, diag::decl_declared_here, Member->getFullName());
   return true;
 }
+
+bool ImplicitCoercionToAnyFailure::diagnoseAsError() {
+  auto E = getAnchor();
+  auto &TC = getTypeChecker();
+  auto &CS = getConstraintSystem();
+
+  auto locator = CS.getConstraintLocator(E, ConstraintLocator::Member);
+  auto resolvedOverload = getResolvedOverload(locator);
```

Comment:
I'd suggest you use `getOverloadChoiceIfAvailable` here.

Reply:
Done!